Description: update
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 lenstoggler (0.1.1-1) unstable; urgency=low
 .
   * source package automatically created by stdeb 0.6.0+git
Author: Andrew King (No comment) <aking1012.com@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- lenstoggler-0.1.1.orig/setup.py
+++ lenstoggler-0.1.1/setup.py
@@ -2,7 +2,7 @@ from setuptools import setup
 
 setup(
     name='LensToggler',
-    version='0.1',
+    version='0.1.1',
     description='Simple App to Toggle Lenses.',
     author='aking1012.com@gmail.com',
     author_email='aking1012.com@gmail.com',
--- lenstoggler-0.1.1.orig/LensToggler.egg-info/PKG-INFO
+++ lenstoggler-0.1.1/LensToggler.egg-info/PKG-INFO
@@ -1,6 +1,6 @@
 Metadata-Version: 1.1
 Name: LensToggler
-Version: 0.1
+Version: 0.1.1
 Summary: Simple App to Toggle Lenses.
 Home-page: http://www.github.com/aking1012/pyeggtut
 Author: aking1012.com@gmail.com
